openrr-command/src/robot_command.rs

```diff
@@ -481,6 +481,35 @@ pub fn load_command_file_and_filter(file_path: PathBuf) -> Result<Vec<String>, O
         .collect())
 }
 
+pub fn robot_command_editor(robot_command: &RobotCommand) -> RobotCommand {
```

Comment:
I don't like this function name. It's better if it is a verb for a function. (ex, `filter_escape_doulbe_quote`)

* Is it possible to make it private?
* Because it is handling `ExecuteCommand`, how about doing this around L365?